### DISCUSSION TITLE
How to parse a multi-column PDF like this SEC 13F document?

### DISCUSSION THREAD

**Discussion by gsbell** [UglyToad/PdfPig#416] [Q&A] [answered]
https://www.sec.gov/files/investment/13flist2021q3.pdf

Basically, I just need to read through this document and parse the columns "CUSIP NO", "ISSUER NAME",  "ISSUER DESCRIPTION", "STATUS", so I can then store them in database for further processing.

I have no idea why they don't provide this data in CSV format.  But that's government efficiency for you.

**Comment by BobLd** [comment, marked as answer]
Hi @gsbell, as a reminder, a pdf is not aware of tables... so you'll have to either find a way to parse it or use one of the libraries below:
- https://github.com/BobLd/tabula-sharp
- https://github.com/BobLd/camelot-sharp

**Comment by gsbell** [comment]
Thank you, I got tabula to do exactly what I needed.
